src/echo/freedom-module.ts

```diff
@@ -2,6 +2,8 @@
 /// <reference path='../logging/logging.d.ts' />
 /// <reference path='../freedom/typings/freedom.d.ts' />
 
+freedom['loggingprovider']().setConsoleFilter(['*:D']);
```

Comment:
It's often useful to adjust the logging level when working on new features or debugging an issue, so this is an example of how to do it.